api/app/models/land_change/tree_cover.py

```diff
@@ -0,0 +1,87 @@
+from typing import Annotated, List, Literal, Optional, Union
+
+from pydantic import Field
+
+from app.models.common.analysis import AnalysisStatus, AnalyticsIn
+from app.models.common.areas_of_interest import (
+    AdminAreaOfInterest,
+    CustomAreaOfInterest,
+    IndigenousAreaOfInterest,
+    KeyBiodiversityAreaOfInterest,
+    ProtectedAreaOfInterest,
+)
+from app.models.common.base import Response, StrictBaseModel
+
+AoiUnion = Union[
+    AdminAreaOfInterest,
+    KeyBiodiversityAreaOfInterest,
+    ProtectedAreaOfInterest,
+    IndigenousAreaOfInterest,
+    CustomAreaOfInterest,
+]
+
+
+ValidTCD = Literal["10", "15", "20", "25", "30", "50", "75", "90"]
+
+
+class TreeCoverAnalyticsIn(AnalyticsIn):
+    aoi: Annotated[AoiUnion, Field(discriminator="type")] = Field(
+        ...,
+        title="AOI",
+        description="AOI to calculate in.",
+    )
+    tcd_threshold: ValidTCD = Field(
```

Comment:
we've been calling it `canopy_cover` across other endpoints

Reply:
Might be time to move the `canopy_cover` literal list to common? 🤷